conf/modules/prepare_references.config

```diff
@@ -77,28 +77,28 @@ process {
     }
 
     withName: '.*PREPARE_REFERENCES:TABIX_PT' {
-        ext.when = { params.target_bed && params.target_bed.endsWith(".gz") }
+        ext.when = { params.target_bed && params.target_bed.endsWith(".gz") && (params.analysis_type == "wes") }
```

Comment:
This should be executed whenever a target bed file is provided, regardless if the analysis type is `wes` or not. Won't this turn off the this process for non wes analysis? This check should take see to that the bed file is always provided in the case of wes https://github.com/nf-core/raredisease/blob/6023bf60ab47100ccc7fdfcc0d412d68cb537ee7/workflows/raredisease.nf#L88

same question for the rest of the changes in this file

Reply:
Yupe! Good catch! Fixed 👍🏻